sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -1,20 +1,23 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2022-09-09)
+This is the first stable release for the Device Update for IoT Hub client library.
 
-### Features Added
-- Added filter to listDeviceClasses device management method
-- Updated description for some methods to be more descriptive and less ambiguous
+### Changes since 1.0.0-beta.3
```

Comment:
This may not het picked up by Automation.
We can make this a Sub header under standard headers of Features added / Other Changes / Breaking Changes etc.